### PR TITLE
fix: fix switch component CSS

### PR DESCRIPTION
**Pull request by burakukula**
Fixing small CSS issue
Screenshot 2022-03-16 at 17 35 42

Screenshot 2022-03-16 at 17 35 47

### CODE REVIEW

File: packages/forma-36-react-components/src/components/Switch/Switch.css [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
     & .Switch__icon--close {
       top: 3px;
-      left: 17px;
+      left: 16px;
```
Comment (massao) [inline packages/forma-36-react-components/src/components/Switch/Switch.css:48]: shouldn't these also be as rem??

File: packages/forma-36-react-components/src/components/Switch/Switch.css [hunk 2]
```diff
@@ -133,5 +133,5 @@
 
 .Switch__icon--close {
   top: 4px;
-  left: 22px;
+  left: 20px;
```
Comment (massao) [inline packages/forma-36-react-components/src/components/Switch/Switch.css:136]: same here

File: packages/forma-36-react-components/src/components/Switch/Switch.css [hunk 3]
```diff
@@ -133,5 +133,5 @@
 
 .Switch__icon--close {
   top: 4px;
-  left: 22px;
+  left: 1,25rem;
```
Comment (massao) [inline packages/forma-36-react-components/src/components/Switch/Switch.css:136]:
```suggestion
  left: 1.25rem;
```

File: packages/forma-36-react-components/src/components/Switch/Switch.css [hunk 4]
```diff
@@ -35,17 +35,17 @@
...
 .Switch--small {
   width: calc(1rem * (28 / var(--font-base-default)));
-  height: calc(1rem * (12 / var(--font-base-default)));
+  height: calc(1rem * (14 / var(--font-base-default)));
 
   & + .Switch__label {
     & .Switch__icon--done {
-      top: 3px;
-      left: 3px;
+      top: 0.15rem;
```
Comment (massao) [inline packages/forma-36-react-components/src/components/Switch/Switch.css:42]: the top isn't needed, if we remove it from the icons it should be centered horizontally

```suggestion
      top: 0.15rem;
```

**Review by Lelith — APPROVED** [review]

**Review by massao — APPROVED** [review]

**Pull request merged**